src/common/policy.js

```diff
@@ -17,12 +17,12 @@ const policy = {
       description: 'Can approve/reject trees',
     },
     {
-      name: 'list_planter',
-      description: 'Can view planters',
+      name: 'list_grower',
```

Comment:
Unfortunately, this policy name value is encoded in records in the `admin_role` table in the database.
We should aim to eventually change it, but it will require some coordination across the deployments.